compio-fs/src/pipe.rs

```diff
@@ -29,8 +34,12 @@ use crate::File;
 /// ```
 pub fn anonymous() -> io::Result<(Receiver, Sender)> {
     let (receiver, sender) = os_pipe::pipe()?;
-    let receiver = Receiver::from_file(unsafe { File::from_raw_fd(receiver.into_raw_fd()) })?;
-    let sender = Sender::from_file(unsafe { File::from_raw_fd(sender.into_raw_fd()) })?;
+    let receiver = Receiver::from_file(File::from_std(unsafe {
```

Comment:
Should we impl `FromRawFd` for `File`?

Reply:
Yes, but syntactically it should use `from_std` here. `from_std` attaches the file to the driver, while `from_raw_fd` should not.

Reply:
Do you mean semantically?

Reply:
Yes:(